File: evalio-ui/src/services/manager/entities/exams.ts
export interface ExamResponse {
    group_id:               string;
    created_at:             Date;
    student_identification: number;
    student_name:           string;
    exam_path:              string;
    template_id:            string;
    period:                 string;
    status:                 string;
    group_name:             string;
}

File: evalio-manager/domain/templates/entities/exams.py
from pydantic import BaseModel
from datetime import datetime
from pytz import timezone

class Exam(BaseModel):
    group_id: str
    created_at: datetime = datetime.now(timezone('America/Bogota'))
    student_identification: int
    student_name: str
    exam_path: str
    template_id: str
    period: str = ""
    status: str
    group_name: str = ""
